src/View/Helper/TimeHelper.php

```diff
@@ -33,6 +34,121 @@ class TimeHelper extends Helper
     use StringTemplateTrait;
 
     /**
+     * The default locale to be used for displaying formatted date strings.
+     *
+     * @var string
+     * @deprecated Use self::setDefaultLocale() and self::getDefaultLocale() instead.
+     */
+    protected $_defaultLocale = null;
+
+    /**
+     * The default format to be used for displaying formatted date strings.
+     *
+     * @var string|null
+     */
+    protected $_defaultFormat = null;
+
+    /**
+     * The \DateTimeZone default output timezone.
+     * See http://php.net/manual/en/timezones.php.
+     *
+     * @var \DateTimeZone|null
+     */
+    protected $_defaultOutputTimezone = null;
+
+    /**
+     * Construct the TimeHelper
+     *
+     * @param \Cake\View\View $View The View this helper is being attached to.
+     * @param array $config Configuration settings for the helper.
+     */
+    public function __construct(View $View, array $config = [])
+    {
+        parent::__construct($View, $config);
+        if (isset($config['defaultLocale'])) {
+            $this->_defaultFormat = $config['defaultLocale'];
+        }
+        if (isset($config['defaultFormat'])) {
+            $this->_defaultFormat = $config['defaultFormat'];
+        }
+        if (isset($config['defaultOutputTimezone'])) {
+            $this->_defaultOutputTimezone = $config['defaultOutputTimezone'];
+        }
+    }
+
+    /**
+     * Gets the default output timezone.
+     *
+     * @return \DateTimeZone|null DateTimeZone object in which the date will be displayed or null.
+     */
+    public function getDefaultOutputTimezone()
+    {
+        return $this->_defaultOutputTimezone;
+    }
+
+    /**
+     * Sets the default output timezone.
+     *
+     * @param string|\DateTimeZone $timezone Timezone string or DateTimeZone object
+     * in which the date will be displayed.
+     * @return $this
+     */
+    public function setDefaultOutputTimezone($timezone)
+    {
+        if ($timezone === null) {
```

Comment:
Lets not have rare use cases :smile: Instead we can try and guide people to not need to do this kind of thing.